e2e/healthcheck.test.ts

```diff
@@ -0,0 +1,10 @@
+import { Client } from "../libs/client";
+import testVars from "./config";
+
+describe("Verify client API healthcheck", () => {
+  it("Shoud return 200 if the client is running", async () => {
+    console.log("API: ", process.env.APP_CLIENT_API_URL);
+    const client = new Client(process.env.APP_CLIENT_API_URL);
+    expect(await client.healthCheck()).toBeTruthy();
+  });
+});
```

Comment:
This test exists in [nightfal_3/test/e2e/protocol/health-and-contracts.test.mjs](https://github.com/EYBlockchain/nightfall_3/blob/master/test/e2e/protocol/health-and-contracts.test.mjs#L30)